packaging/docker/Dockerfile

```diff
@@ -101,4 +101,8 @@ RUN \
 ENV NETDATA_PORT 19999
 EXPOSE $NETDATA_PORT
 
+# Assume everything works fine when netdata presents metrics api enpoint.
+HEALTHCHECK --interval=3s --timeout=5s \
+    CMD [ "curl", "-f", "localhost:$NETDATA_PORT/api/v1/allmetrics" ]
```

Comment:
This is an expensive call. It dumps all the metrics to text so some CPU will be used.
A lighter one is `/api/v1/registry?action=hello` or the new one `/api/v1/info`.